lib/helpers/cli.js

```diff
@@ -130,7 +132,7 @@ export function parseArgv(_argv) {
     .usage('$0 [options] [files..]')
     .options(specifiedOptions)
     .help()
-    .version();
+    .version(require('../../package.json').version);
```

Comment:
This seems to indicate there is a yargs bug in retrieving the version, perhaps when ESM is present? Is there an active yargs issue for this? I'd prefer it was fixed there. If we are going to add a workaround, we would at least need to comment to explain why it's needed.

Reply:
You're suspicion is correct!: https://github.com/yargs/yargs/issues/2122

Reply:
I've verified https://github.com/yargs/yargs/pull/2123 fixes the issue, so hopefully that will go in soon!